src/zen/websocket_server.cpp

```diff
@@ -209,6 +212,22 @@ class WsHandler
         wsq.push(wse);
     }
 
+    void sendBlockHeaders(UniValue headers, WsEvent::WsMsgType msgType, std::string clientRequestId = "")
```

Comment:
'UniValue headers' could be made 'const UniValue& headers' in order not to copy the input object